docs/troubleshooting.md

```diff
@@ -46,6 +48,41 @@ This is typically encountered when you attempt to install BKPR with a DNS zone (
 
 ![Azure SP Conflict](images/azure-sp-conflict.png)
 
+## Troubleshooting BKPR ingress
```

Comment:
I think this section should be about where to look when things are not working as expected. For example if a user accesses the prometheus dashboard and gets a self-signed certificate message, then something is probably not working as desired. This document should be helpful to the user to debug / understand what is going on and how to try and troubleshoot it.